.gitignore

```diff
@@ -1 +1,3 @@
-exampleSite/public
+exampleSite/public/
+exampleSite/resources/
+docs/css/main*
```

Comment:
See my comment in https://github.com/victoriadotdev/hugo-theme-introduction/pull/137